packages/documentation/src/stories/foundation/accessibility/accessibility.docs.mdx

```diff
@@ -12,6 +12,22 @@ import liveRegionBadAngularSample from './liveregion-bad-angular.sample.html?raw
   TODO: write accessibility page
 </div>
 
+The following article of the Swiss postal law is an official ruling concerning accessibility issues.
+
+<figure>
+  <blockquote class="blockquote" lang="de">
+    <q>Die Postdienste müssen so angeboten werden, dass Menschen mit Behinderungen
+sie in qualitativer, quantitativer und wirtschaftlicher Hinsicht unter vergleichbaren
+Bedingungen wie Menschen ohne Behinderungen beanspruchen können. Insbesondere müssen:<br/>
+a. die Zugangspunkte den Bedürfnissen von Kundinnen und Kunden mit sensorischen oder Bewegungsbehinderungen entsprechen;<br/>
+b. Sendungen von und für Menschen mit Sehbehinderungen portofrei versandt
+werden können.</q>
+  </blockquote>
```

Comment:
After reading the quote, half of it does not make sense for digital products. Also, the `q` tag is for inline quotations.

```suggestion
  <blockquote class="blockquote" lang="de">
    <p>Die Postdienste müssen so angeboten werden, dass Menschen mit Behinderungen
sie in qualitativer, quantitativer und wirtschaftlicher Hinsicht unter vergleichbaren
Bedingungen wie Menschen ohne Behinderungen beanspruchen können.</p>
  </blockquote>
```